sklearn/tree/tree.py

```diff
@@ -108,6 +108,20 @@ def __init__(self,
         self.class_weight = class_weight
         self.presort = presort
 
+    def get_depth(self):
+        """Returns the depth of the decision tree.
+
+        The depth of a tree is defined as the maximum depth of its leaves.
```

Comment:
Maybe more explicit: "maximum number of nodes between the root and any leaf, inclusive of endpoints."

Reply:
It's not inclusive of both endpoints though. The depth is correctly reported as 1 for a tree with a root and two leaves.

Reply:
Good that I clarified then. "The number of branching nodes from the root to the leaf", then?

Reply:
That may be more confusing, wouldn't it? How about "maximum distance between the root and any leaf"? I had it changed, the push didn't get through, didn't notice.